### DISCUSSION TITLE
Gli albori del prestigioso Osservatorio Star Citizen

### DISCUSSION THREAD

**Discussion by pdonadeo** [pdonadeo/e-uscito-joypad#25] [General]
Guardando nel nostro database, che prende sempre più forma, si possono già ora scoprire cose non immediatamente evidenti. Per esempio possiamo sapere a quando risale la prima citazione a **Star Citizen**.

Si tratta dell'episodio numero 4, pubblicato nell'ormai lontano 31 Gennaio 2020.

Primo commento di @zampale: _«Qui siamo al limite del Tubo Tucker»_

😂

**Comment by agofa** [comment]
Ah ah ah ah, nulla sarà dimenticato

**Comment by pdonadeo** [comment]
Ho modificato leggermente il DB: ho aggiunto alle modalità di citazione, oltre a "Chiacchiera libera", "Recensione" e "Consiglio" anche una voce specifica "Osservatorio Star Citizen".


![image](https://user-images.githubusercontent.com/242049/170826382-8aebdba1-a1a0-4b93-afa3-c6ceeeec173f.png)

**Reply by agofa** [reply]
> Ho modificato leggermente il DB: ho aggiunto alle modalità di citazione, oltre a "Chiacchiera libera", "Recensione" e "Consiglio" anche una voce specifica "Osservatorio Star Citizen".

Usato poco fa nell'ep 48.

**Comment by zampale** [comment]
Questa la uso già oggi, sappiatelo.

Tutti preziosissimi.
Il 28 mag 2022, 11:50 +0200, Paolo Donadeo ***@***.***>, ha scritto:
